### DISCUSSION TITLE
Release file(s) cannot uploaded

### DISCUSSION THREAD

**Discussion by TheWithCommands** [community/community#40734] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

When releasing a Release, may need to upload files. However, if try to upload, the upload window will display "Something went really wrong, and we can't process that file.", and 404 Not Found error will appear on the console.
![捕获](https://user-images.githubusercontent.com/67952504/205474636-908a8447-5b21-4cbf-8f10-c433f577d4c9.PNG)

**Comment by jfullstackdev** [comment, marked as answer]
I did not have any problem upon trying, even for other type of files

![temptemptemp](https://user-images.githubusercontent.com/47092464/205498156-4042f25e-f43e-4cda-9a92-a40057c29f29.png)

is that file more than 25MB ? what kind of file is that ? try switching connection too, sometimes one network causes errors on GitHub, though ( and as far as I know ) GitHub never filters connection requests from users, using VPN usually solves the problem

do you have GitHub extensions ? try logging in in Incognito mode, maybe that extension is preventing it

**Reply by TheWithCommands** [reply]
I tried to clear the browser cache, and now I can upload normally.
